lib/src/revset.rs

```diff
@@ -3592,11 +3592,11 @@ mod tests {
         let id_2 = CommitId::from_hex("222222");
         let id_3 = CommitId::from_hex("333333");
         let id_4 = CommitId::from_hex("444444");
-        index.add_commit_data(id_0.clone(), new_change_id(), &[]);
-        index.add_commit_data(id_1.clone(), new_change_id(), &[id_0.clone()]);
-        index.add_commit_data(id_2.clone(), new_change_id(), &[id_1.clone()]);
-        index.add_commit_data(id_3.clone(), new_change_id(), &[id_2.clone()]);
-        index.add_commit_data(id_4.clone(), new_change_id(), &[id_3.clone()]);
+        index.add_commit_data(id_0.clone(), new_change_id_deprecated(), &[]);
+        index.add_commit_data(id_1.clone(), new_change_id_deprecated(), &[id_0.clone()]);
+        index.add_commit_data(id_2.clone(), new_change_id_deprecated(), &[id_1.clone()]);
+        index.add_commit_data(id_3.clone(), new_change_id_deprecated(), &[id_2.clone()]);
+        index.add_commit_data(id_4.clone(), new_change_id_deprecated(), &[id_3.clone()]);
```

Comment:
This test doesn't seem to care the they're different, so we can probably pass the same value to all. That value can be `ChangeId::from_bytes(&[0; 16])`. It can even be `ChangeId::from_hex("123456")` or whatever you prefer since `MutableIndex` doesn't actually care that the `ChangeId` has a certain size.

Reply:
I haven't looked at it at all, just let rust-analyzer rename the function. I'll look at it.

Reply:
My eyes glaze over a little from all these tests. Is it OK to leave this for another day?

Reply:
Sure, no problem.